src/Layers/xrRender/D3DXRenderBase.h

```diff
@@ -46,7 +46,7 @@ class D3DXRenderBase : public IRender, public pureFrame
 
     // Runtime structures
     xr_vector<R_dsgraph::mapNormalVS::value_type *> nrmVS;
-#ifndef USE_DX9
+#if defined(USE_DX11) || defined(USE_OGL) //USE_DX9
```

Comment:
Many modern graphics APIs have geometry shaders, so, let's better to leave it as is:
```suggestion
#ifndef USE_DX9
```